pxr/base/tf/fileUtils.cpp

```diff
@@ -106,8 +115,7 @@ Tf_HasAttribute(
     }
 
     // Read symlinks until we find the real file.
-    return Tf_HasAttribute(TfReadLink(path.c_str()), resolveSymlinks,
-                           attribute, expected);
+    return Tf_HasAttribute(linkPath, resolveSymlinks, attribute, expected);
```

Comment:
It's a bit tricky to tell that linkPath is non-empty from reading the code; it'd be great if that could be made more obvious

Reply:
Is an additional comment around line 117 sufficient? Something like:

// At this point we know (attribs & FILE_ATTRIBUTE_REPARSE_POINT) != 0
// or we would have returned in the if block above. This means linkPath will
// be holding the result of calling TfReadLink(path.c_str()). The code is separated
// in this way to avoid calling TfReadLink twice.